views/pages/camps/index_admin.jade

```diff
@@ -56,7 +56,7 @@ block content
                             //- th.admin-edit=t(t_prefix+'stats.edit')
                             //- th.admin-remove=t(t_prefix+'stats.remove')
                     tbody
-                        tr(ng-repeat="camp in camps | filter: searchCamp | orderBy: orderCamps")
```

Comment:
As discussed, this breaks textual search functionality

Reply:
fixed